packages/exchange/src/exchange/utils.py

```diff
@@ -1,7 +1,7 @@
 import inspect
 import uuid
 from importlib.metadata import entry_points
-from typing import get_args, get_origin
+from typing import Literal, get_args, get_origin, Any, List, Tuple, Dict, Union
```

Comment:
some of these are [deprecated](https://docs.python.org/3/library/typing.html#deprecated-aliases) now so i think we should use those instead